src/components/blogContent/BlogContent.tsx

```diff
@@ -0,0 +1,33 @@
+import { makeStyles, TextField, Typography } from "@material-ui/core";
+import React from "react";
+const useStyles = makeStyles((theme) => ({
+  "padding-md": {
```

Comment:
one idea of MUI is to style the components as they're written so instead of writing text names for these CSS properties, we can instead call this `FormSection` or something like that and the following can be called `FormField`.